src/streamlink/plugins/bongacams.py

```diff
@@ -138,13 +129,18 @@ def _get_streams(self):
                                        self._get_stream_uid(stream_source_info['userData']['username'])),
             # Multiple args with same name not supported.
             # Details: https://github.com/streamlink/streamlink/issues/321
-            "conn" : "S:{username} --conn=S:{access_key} --conn=B:0 --conn=S:{data_key}".format(
+            "conn": "S:{username} --conn=S:{access_key} --conn=B:0 --conn=S:{data_key}".format(
                 username=stream_source_info['userData']['username'],
                 access_key=stream_source_info['localData']['NC_AccessKey'],
                 data_key=stream_source_info['localData']['dataKey']
             )
         }
 
+        subprocess_errorlog_path = self.session.options.options.get('subprocess-errorlog-path')
```

Comment:
`self.session.logger.level` should have the level. There is no method to map level number to name, or any constants for levels. If you do want to compare log levels I would definitely add some constants to the logging module. 

The logging module is not very good, I'm not sure why the standard python logging library is not used as it's far superior. It's on my TODO list to change all the logging to use python logging, but I haven't got round to it yet :-) Now that I think about it, I should really spend the time and do it...